docs/ai-sdk-v6-improvements.md

```
@@ -0,0 +1,282 @@
+
+// Applied to each model when isDevelopmentEnvironment is true
+wrapLanguageModel({
+  model: openai.languageModel("gpt-5-mini"),
+  middleware: isDevelopmentEnvironment ? devToolsMiddleware() : [],
+}),
+```
+
+### Usage
+1. Start your dev server: `pnpm dev`
+2. In a separate terminal, run: `npx @ai-sdk/devtools`
+3. Open http://localhost:4983 to inspect:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)